.vscode/launch.json

```diff
@@ -29,6 +30,7 @@
                 "10000",
                 "--dart-define=LENRA_SERVER_URL=${env:LENRA_SERVER_URL}",
                 "--dart-define=LENRA_BASIC_AUTH=${env:LENRA_BASIC_AUTH}",
+                "--dart-define=SENTRY_CLIENT_DSN=test",
```

Comment:
Why define `test` as the Sentry client DSN ?

Reply:
From this Pr https://github.com/lenra-io/backoffice/commit/adc0049974554f9a115ca4cab15d36e4395bdd9a we need SENTRY_CLIENT_DSN as environment variable that I defined as test but we can define as empty string or something else.